gensim/test/test_sklearn_integration.py

```diff
@@ -107,5 +108,55 @@ def testPipeline(self):
         score = text_lda.score(corpus, data.target)
         self.assertGreater(score, 0.50)
 
+class TestSklearnLSIWrapper(unittest.TestCase):
+    def setUp(self):
+        self.model = SklearnWrapperLsiModel(id2word=dictionary, num_topics=2)
+        self.model.fit(corpus)
+
+    def testPrintTopic(self):
+        topic = self.model.print_topics(2)
```

Comment:
@tmylk My rationale for having this test was basically to serve as a sanity check after `setUp()` prepares the test fixture. Since such a test was present among the unit tests for the sklearn wrapper written for LDA model as well, I decided to include it here too. Please let me know if we should drop this test.